Solutions/ARGOSCloudSecurity/Analytic Rules/ExploitableSecurityIssues.yaml

```diff
@@ -0,0 +1,45 @@
+id: a9bf1b8c-c761-4840-b9a8-7535ca68ca28
+name: ARGOS Cloud Security - Exploitable Cloud Resources
+description: Exploitable Cloud Security Issues are ones that expose cloud resources   to the internet and allow initial access to your environment.
+requiredDataConnectors:
+  - connectorId: ARGOSCloudSecurity
+    dataTypes:
+      - ARGOS_CL
+kind: Scheduled
+version: 1.0.0
+severity: High
+query: |
+  ARGOS_CL | where exploitable_b
```

Comment:
I assume the intent is to throw an alert for every time exploitable_b column is true, but want to confirm. ;)

Reply:
Yes, that's correct, should we be more explicit here with `ARGOS_CL | where exploitable_b = true` ?

Reply:
@davidobrien1985 - Not required, but I do think for users of the query it does make it more explicit, so I think preferred.